extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -767,19 +767,19 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
     if (!newContacts.length) {
       return;
     }
-    const toLookupNoPubkeys: Contact[] = [];
+    const toLookupNoPubkeys = new Set<EmailProviderContact>();
     for (const input of newContacts) {
-      const contact = await ContactStore.obj({ email: input.email, name: input.name });
-      const [storedContact] = await ContactStore.get(undefined, [contact.email]);
+      // todo: create and use a lighter method in ContactStore that doesn't return any keys?
+      const storedContact = await ContactStore.getOneWithAllPubkeys(undefined, input.email);
       if (storedContact) {
```

Comment:
I think current method is ok, unless we do this a lot. That way, fewer methods to maintain.